src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/InputHelper.java

```diff
@@ -30,4 +46,103 @@ public static InputScannedDoc getSampleInputDocument(String suffix) {
     public static InputScannedDoc getSampleInputDocument() {
         return getSampleInputDocument("");
     }
+
+    public static CaseDetails caseDetails(SampleCase originalCase) {
+        return new CaseDetails("1234567890", "some_type", originalCase);
+    }
+
+    public static SampleCase sampleCase(List<Item<ScannedDocument>> scannedDocuments) {
+        return new SampleCase(
+            "legacy-id",
+            "first-name",
+            "last-name",
+            "date-of-birth",
+            "contact-number",
+            "email",
+            new Address("a", "b", "c", "d", "e", "f", "g"),
```

Comment:
Worth replacing these letters with values that would tell what field they represent.